### ISSUE TITLE
Set-NexusUserPassword reports "Unsupported Media Type"

### ISSUE THREAD

**Issue by Windos** [steviecoaster/NexuShell#9]
When trying to set a password using `Set-NexusUserPassword`, I'm getting an error related to the media type being unsupported.

```powershell
Set-NexusUserPassword -Username admin -NewPassword ("Sausage2021" | ConvertTo-SecureString -AsPlainText -Force)
```

```output
Invoke-RestMethod : The remote server returned an error: (415) Unsupported Media Type.
At C:\Program Files\WindowsPowerShell\Modules\NexuShell\1.0.0\NexuShell.psm1:106 char:9
+         Invoke-RestMethod @Params
+         ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

**Comment by Windos** [comment]
Digging in a little and there's two things here:

First, the content type needs to be `text/plain`. This could either go on the `Invoke-Nexus` call in the function, or the `BodyAsString` section inside `Invoke-Nexus` could override it?

Second, the function currently takes in a secure string, but doesn't convert it to plaintext before sending it through to the API. This means the password is actually being changed to "System.Security.SecureString" (which is the result of using a securestring's ToString method.

**Comment by steviecoaster** [comment]
This is merged in #10
